tests_login_with_email_address/test_invalid_login_details.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from playwright.sync_api import expect
+
+
+def test_blank_password_and_user(page):
+    # Go to https://login.pracuj.pl/
+    page.goto("https://login.pracuj.pl/")
+    # Click [data-test="button-login"]
+    page.locator("[data-test=\"button-login\"]").click()
+
+    email = page.locator("[data-test=\"input-email\"]")
+    expect(email).to_contain_text(" ")
+    assert page.inner_text(
+        "._1Y5MF > div:nth-of-type(1) ._1q5DA") == r"Wypełnij to pole.", f"Actual value: {page.inner_text('._1Y5MF > div:nth-of-type(1) ._1q5DA')}"
+
+    password = page.locator("[data-test=\"input-password\"]")
+    expect(password).to_contain_text(" ")
+    assert page.inner_text(
+        "._1Y5MF > div:nth-of-type(2) ._1q5DA") == r"Wypełnij to pole.", f"Actual value: {page.inner_text('._1Y5MF > div:nth-of-type(2) ._1q5DA')}"
```

Comment:
It's not a good approach to build locators based on `._1Y5MF` because it's some kind of obfuscation and on new version of an application, probably this will not work. Try to build locators based on something else, like general HTML tags (like div, label, etc.) and some attributes that have value based on natural language.